### ISSUE TITLE
Add a `show_seconds` property

### ISSUE THREAD

**Issue by danirabbit** [elementary/granite#208]
I want to use TimePicker in Photos but I need to handle seconds. We should add a property to TimePicker that allows us to show seconds.

It would also be convenient to have a constructor like `new Granite.TimePicker.with_seconds ()`

**Comment by FriendOfEntropy** [comment]
I may have missed it, but I didn't see any labels such as "suitable for new contributors" on any of the issues.

Unless I'm vastly underestimating complexity here, if no one has claimed this one yet, I'd like to take a crack at it.

**Comment by danrabbit** [comment]
@FriendOfEntropy go for it, no need to ask :)

**Comment by FriendOfEntropy** [comment]
Thanks...didn't know if there was a mechanism to keep people from duplicating work on the same issue unbeknownst to each other :)

**Comment by FriendOfEntropy** [comment]
PR created, hope I covered everything.

**Comment by FriendOfEntropy** [comment]
Hoping someone can review my small refactorings in [PR 213](https://github.com/elementary/granite/pull/213).  As discussed in closed [PR 211](https://github.com/elementary/granite/pull/211), I have followed the directions to divide into two PRs so the diffs on each of the PRs will be easier.  213 is the first, and once that is approved and merged into the base, I'll create a new PR with the show_seconds functionality which will hopefully be easier to evaluate on the diff.  Thanks
